openmetadata-spec/src/main/resources/json/schema/entity/services/connections/metadata/atlasConnection.json

```diff
@@ -39,22 +39,10 @@
       "description": "entity types of the data source.",
       "type": "string"
     },
-    "serviceType": {
-      "description": "service type of the data source.",
-      "type": "string"
-    },
     "atlasHost": {
       "description": "Atlas Host of the data source.",
       "type": "string"
     },
-    "dbService": {
-      "description": "source database of the data source.",
-      "type": "string"
-    },
-    "messagingService": {
-      "description": "messaging service source of the data source.",
-      "type": "string"
-    },
     "database": {
```

Comment:
also I'm not sure if we are using `database` field as well, lets review all the fields are remove those which are not required or relevant anymore!